external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaDataConsumerSuite.scala

```diff
@@ -105,7 +105,17 @@ class KafkaDataConsumerSuite extends SharedSQLContext with PrivateMethodTester {
           val headers = record.headers().toArray.map(header => (header.key(), header.value())).toSeq
           (value, headers)
         }
-        data === rcvd
+        data zip rcvd foreach { case (expected, actual) =>
```

Comment:
Let's avoid postfix notation -- `data.zip(rcvd).foreach` I think it generates a compiler warning in some cases so we have avoided it.

Reply:
@srowen Got it. Here is the fix.